book/src/install.md

```diff
@@ -149,7 +151,7 @@ cargo install --path helix-term --locked
 
 This command will create the `hx` executable and construct the tree-sitter
 grammars in the local `runtime` folder. To build the tree-sitter grammars requires
-a c++ compiler to be installed, for example `gcc-c++`.
+a c++ compiler to be installed, for example `gcc-c++` or `clang`.
```

Comment:
This bit has now been addressed in https://github.com/helix-editor/helix/pull/6809